terraform/table_terraform_output.go

```diff
@@ -87,42 +89,72 @@ type terraformOutput struct {
 }
 
 func listOutputs(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
-	// The path comes from a parent hydate, defaulting to the config paths or
+	// The path comes from a parent hydrate, defaulting to the config paths or
 	// available by the optional key column
-	data := h.Item.(filePath)
-	path := data.Path
+	pathInfo := h.Item.(filePath)
+	path := pathInfo.Path
 
 	// Return if the path is a TF plan path
-	if data.IsTFPlanFilePath {
+	if pathInfo.IsTFPlanFilePath {
 		return nil, nil
 	}
 
-	combinedParser, err := Parser()
-	if err != nil {
-		plugin.Logger(ctx).Error("terraform_output.listOutputs", "create_parser_error", err)
-		return nil, err
-	}
-
 	content, err := os.ReadFile(path)
 	if err != nil {
 		plugin.Logger(ctx).Error("terraform_output.listOutputs", "read_file_error", err, "path", path)
 		return nil, err
 	}
 
-	var tfOutput terraformOutput
+	var docs []model.Document
+
+	// Check if the file contains TF state
+	if pathInfo.IsTFStateFilePath {
```

Comment:
Does the parser allow us to use a single parser for JSON and Terraform? Do we separate do we can have better logging messages?

Reply:
No, There are different parsers for JSON and Terraform. Could you please let me know which logging message needs to be updated, the error or the comment?